packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -78,7 +59,14 @@ const GitLabScopingSearchResults = (props: Props) => {
   >(
     graphql`
       fragment GitLabScopingSearchResults_query on Query
-        @argumentDefinitions(cursor: {type: "String"}, count: {type: "Int", defaultValue: 50})
+        @argumentDefinitions(
+          projectsFirst: {type: "Int", defaultValue: 5}
+          issuesFirst: {type: "Int", defaultValue: 25}
```

Comment:
We can't add a connection to issues as connections can't appear inside plural fields like `projects`. So, I've added the connection to the projects, and I'm getting the first 25 issues in each project.

If the issue that the user is looking for isn't included within that list, they'll be able to search for the title or description to be able to find it.